lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -1280,6 +1280,149 @@
   }
 }
 
+// view - discussions management
+// --------------------
+.instructor-dashboard-wrapper-2 section.idash-section#discussions_management {
+
+  // cohort management
+  .form-submit {
+    @include idashbutton($uxpl-blue-base);
+    @include font-size(14);
+    @include line-height(14);
+    margin-right: ($baseline/2);
+    margin-bottom: 0;
+    text-shadow: none;
+  }
+
+  .cohort-management-file-upload {
```

Comment:
I will go in and remove what I can, but I avoided making major adjustments to this file because I knew we were going to update the ui in another PR.